handbook/benefits.md

```diff
@@ -0,0 +1,41 @@
+## Why Fleet?
+
+We believe device management should be safe, easy, and transparent to end users.
+
+### The premier osquery fleet manager.
+Fleet makes it easier to query and track your servers, containers, and laptops. It extends osquery to answer questions about multiple devices at the same time and provides log streams that enable automated threat detection. You can run it on your own hardware or deploy it in any cloud.
+
+#### Realtime visibility
+- Share with any team (ops, sec, helpdesk, etc)
+- Unmetered access to user seats
+- Intuitive user interface (L1-friendly)
+- Verify remediations were actually successful
+- Make compliance goals highly visible
+- Track progress towards compliance goals together
+- Auto-ticket any compliance regressions
+- Reliable, efficient source of truth
+
+#### Get flexible
+- Receive data from any device
+- Send data anywhere it needs to go
+- Express and track custom compliance goals
+- Integrate existing business processes
+- Play nice with internal tools and requirements
+- Deploy anywhere
+
+#### Developer productivity
+- Programmable (REST API, ±gRPC)
+- Command line tool (fleetctl)
+- "Infrastructure as code"-ready (YAML)
+- All UI functionality accessible via APIs
+- Focus on value, not boilerplate
+- Skip reinventing the wheel
+- Ad hoc audits (live queries answer in seconds)
+- 100% source code available
+
+### Unlock efficiencies
```

Comment:
@noahtalerman is the `###` vs. `####` intentional?

Reply:
No, thank you for catching that. Fixed now.